src/data-loaders/RelationalDataLoader.js

```diff
@@ -94,12 +94,40 @@ const convertPostgresSQLResponseToColumnMetaData = (rows) => {
       label: row.name,
       name: row.name,
       type: row.dataTypeID,
-      typeName,
+      typeName: typeName ? typeName.toUpperCase() : 'UNKOWN',
     };
   });
 };
+
+const injectVariables = (statement, variableMap) => {
+  const result = Object.keys(variableMap).reduce((statmnt, key) => {
+    if (
+      variableMap[key] === null ||
+      variableMap[key] === false ||
+      variableMap[key] === true
+    ) {
+      return statmnt.replace(key, `${variableMap[key]}`);
```

Comment:
also, let's please use replaceAll instead. Just incase the variable occurs more than once

Reply:
@katesclau i this you need to use `replaceAll` here too.